### PR TITLE
fix: add xpub not working

### PR DESCRIPTION
**Pull request by lukicenturi**
- [x] Add xpub not working
- [x] Xpub not scrambled

### CODE REVIEW

File: frontend/app/src/composables/blockchain/index.ts [hunk 1]
```diff
@@ -269,12 +269,14 @@ export function useBlockchains() {
     }
 
     const registeredAddresses: string[] = [];
-    const failedPayload: AccountPayload[] = [];
+    const failedPayload: (AccountPayload | XpubAccountPayload)[] = [];
```
Comment (kelsos) [inline frontend/app/src/composables/blockchain/index.ts:272]: can you check the changes from the account's branch for this and pull them over?

File: frontend/app/src/composables/api/blockchain/accounts.ts [hunk 2]
```diff
@@ -66,8 +66,8 @@ export function useBlockchainAccountsApi() {
     const url = !Array.isArray(pay)
       ? `/blockchains/${chain}/xpub`
       : `/blockchains/${chain}/accounts`;
-    const payload = (Array.isArray(pay)) ? { accounts: pay } : pay;
-    return performAsyncQuery(url, nonEmptyProperties(payload));
+    const payload = (Array.isArray(pay)) ? { accounts: pay } : { ...pay, ...pay.xpub };
```
Comment (kelsos) [inline frontend/app/src/composables/api/blockchain/accounts.ts:69]: Same here there are changes in the `accounts` branch can you also check then and pull them instead?

Comment (lukicenturi) [reply frontend/app/src/composables/api/blockchain/accounts.ts]: Updated

File: frontend/app/src/composables/scramble.ts [hunk 3]
```diff
@@ -10,20 +10,22 @@ export function useScramble() {
 
   const scrambleData = logicOr(scrambleSetting, logicNot(shouldShowAmount));
 
-  const scrambleHex = (hex: string): string => {
+  const scrambleAddress = (address: string): string => {
```
Comment (lukicenturi) [inline frontend/app/src/composables/scramble.ts:13]: Also I renamed this function to make it more clear.

File: frontend/app/src/utils/truncate.ts [hunk 4]
```diff
@@ -7,6 +9,22 @@ export const truncationPoints: Record<string, number> = {
   '2xl': 40,
 };
 
+export function findAddressKnownPrefix(address: string) {
+  const truncatePrefixExceptions = [
+    '0x',
+    ...Object.values(XpubPrefix),
```
Comment (lukicenturi) [inline frontend/app/src/utils/truncate.ts:15]: Updated @kelsos

**Pull request merged**